packages/modules/order/src/services/order-module-service.ts

```diff
@@ -2753,10 +2753,55 @@ export default class OrderModuleService<
     }
 
     await this.orderService_.update(
+      orderIds.map((id) => {
+        return {
+          id,
+          status: OrderStatus.COMPLETED,
+        }
+      }),
```

Comment:
question: Do we have a `this.update()` that accepts a selector?

Reply:
yes, but that will also call this same internal service.
do you think we should change it?

Reply:
I am fine keeping it as is, just thought it would read a bit simpler. Nothing to address now